### DISCUSSION TITLE
Using Global Data File Variables To Populate .njk Meta <Title>?

### DISCUSSION THREAD

**Discussion by chrisativonne** [11ty/eleventy#2018] [Q&A]
Is it possible to use variables from my global title file in the <title> meta?

Example: I have a company.json file in _data that includes my company name, tagline, phone number. I'd like to be able to use these in the title meta of my content pages.

e.g.
---
layout: default
title: '{{ company.name }} | {{ company.tag_line }} | {{ company.telephone }}'
---

When I add this the title in browser literally shows as {{ company.name }} | {{ company.tag_line }} | {{ company.telephone }} instead of the values from the file. 

If instead I remove the quotes I get this on restarting 11ty
`YAMLException` was thrown:
    YAMLException: can not read a block mapping entry; a multiline key may not be an implicit key at line 4, column 1:

If I test just one of the objects by itself it shows up as a literal {{ company.name }}.

Can this be done?

**Comment by pdehaan** [comment]
You probably want
https://www.11ty.dev/docs/data-computed/
